src/App.tsx

```diff
@@ -1,14 +1,65 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+
+import { getTodos } from './api';
+
+const filterTodos = (todos: Todo[], show: string, query: string): Todo[] => {
```

Comment:
I don't quite like the naming on `show` - the meaning is not clear

better options: `completed`, `status`, etc

In addition - you probably should limit possible input values by the type, for instance:
```suggestion
const filterTodos = (todos: Todo[], todoStatus: 'completed'| 'pending' | 'all' , query: string): Todo[] => {
```